components/match2/commons/starcraft_starcraft2/match_summary_starcraft_ffa.lua

```diff
@@ -9,470 +9,132 @@
-}
+---@param match StarcraftMatchGroupUtilMatch
+---@return Widget
+function StarcraftMatchSummaryFfa._schedule(match)
+	if StarcraftMatchSummaryFfa._gamesHaveDates(match) then
+		return MatchSummaryWidgets.GamesSchedule{games = match.games}
+	end
+	return MatchSummaryWidgets.MatchSchedule{match = match}
+end
 
-local MATCH_STANDING_COLUMNS = {
-	{
-		class = 'cell--status',
-		--iconClass = 'fas fa-hashtag',
-		show = {
-			value = function(match)
+---@param match StarcraftMatchGroupUtilMatch
+---@return boolean
+function StarcraftMatchSummaryFfa._gamesHaveDates(match)
+	local dates = Array.map(match.games, Operator.property('date'))
+	return Array.any(dates, function(date) return date ~= match.date end)
+end
+
+---@param columns table[]
+---@return table[]
+function Parser.adjustMatchColumns(columns)
+	return Array.map(columns, function(column)
+		if column.id == 'status' then
+			column.show = function(match)
 				return match.finished and #Array.filter(match.opponents, function(opponent)
 					return Logic.readBool(opponent.advances)
 				end) > 1
 			end
-		},
-		row = {
-			value = function (opponent, idx)
-				local statusIcon = Logic.readBool(opponent.advances) and STATUS_ICONS.advances or STATUS_ICONS.eliminated
-				return mw.html.create('i')
-					:addClass(statusIcon)
-			end,
-		},
-	},
-	{
-		class = 'cell--rank',
-		iconClass = 'fas fa-hashtag',
-		header = {
-			value = 'Rank',
-		},
-		row = {
-			value = function (opponent, idx)
-				local place = opponent.placement ~= -1 and opponent.placement or idx
-				local worstPlace = opponent.extradata.worstPlace
-				local icon, color = StarcraftMatchSummaryFfa._getTrophy(place)
-				return mw.html.create()
-						:tag('i'):addClass('panel-table__cell-icon'):addClass(icon):addClass(color):done()
-						:tag('span'):wikitext(StarcraftMatchSummaryFfa._displayRank(place, worstPlace)):done()
-			end,
-		},
-	},
-	{
-		class = 'cell--team',
-		iconClass = 'fas fa-users',
-		header = {
-			value = 'Participant',
-		},
-		row = {
-			value = function (opponent, idx)
-				return StarcraftMatchSummaryFfa._displayOpponent(opponent)
-			end,
-		},
-	},
-	{
-		class = 'cell--total-points',
-		iconClass = 'fas fa-star',
-		header = {
-			value = 'Total Points',
-			mobileValue = 'Pts.',
-		},
-		show = {
-			value = function(match)
-				return not Logic.readBool(match.noScore)
-			end
-		},
-		row = {
-			value = function (opponent, idx)
-				return OpponentDisplay.InlineScore(opponent)
-			end,
-		},
-	},
-	game = {
-		{
-			class = 'panel-table__cell__game-placement',
-			iconClass = 'fas fa-trophy-alt',
-			header = {
-				value = 'Place',
-				mobileValue = 'P.',
-			},
-			row = {
-				class = function (opponent)
-					return PLACEMENT_BG[opponent.placement]
+			column.row = {
+				value = function (opponent, idx)
+					local statusIcon = Logic.readBool(opponent.advances)
+						and BaseMatchSummary.STATUS_ICONS.advances
+						or BaseMatchSummary.STATUS_ICONS.eliminated
```

Comment:
Couldn't this logic be moved to MGI? if advances -> set the bg
also these two icons here don't exist